src/main/webapp/view/transcript/transcript.js

```diff
@@ -149,18 +151,18 @@ function removeBold(transcriptLineLiElement) {
   transcriptLineLiElement.classList.remove(BOLD_FONT_WEIGHT);
 }
 
-/** Returns true if`transcriptLineLiElement` is bolded. */
+/** Returns true if `transcriptLineLiElement` is bolded. */
 function isBolded(transcriptLineLiElement) {
   return transcriptLineLiElement.classList.contains(BOLD_FONT_WEIGHT);
 }
 
 /**
- * Returns true if `currentTimeMs` is within the time range for
- * the current transcript line.
+ * Returns true if `timeMs` is within the time range of
+ * 'transcriptLine'
  */
-function isWithinCurrentTimeRange(currentTimeMs) {
-  return currentTranscriptLine.startTimestampMs <= currentTimeMs &&
-      currentTimeMs <= currentTranscriptLine.endTimestampMs;
+function isWithinTimeRange(timeMs, transcriptLine) {
```

Comment:
This method is being edited in another PR? Is this PR going to be merged before?

Reply:
Great!